### PR TITLE
LPS-68220 Implementation of OpenAM versions for OpenSSO authentication

### PR DESCRIPTION
**Pull request by martamedio**

### CODE REVIEW

**Comment by martamedio** [comment]
ci:test:sf

**Comment by martamedio** [comment]
ci:test

**Comment by martamedio** [comment]
ci:reopen

**Comment by martamedio** [comment]
ci:test:relevant

**Comment by martamedio** [comment]
ci:test:sf

File: modules/apps/portal-security-sso/portal-security-sso-opensso-impl/src/main/java/com/liferay/portal/security/sso/opensso/internal/OpenSSOImpl.java [hunk 1]
```diff
@@ -401,8 +337,6 @@ public void setCookieProperty(
 	private static final String _GET_COOKIE_NAMES =
 		"/identity/getCookieNamesToForward";
```
Comment (topolik) [inline modules/apps/portal-security-sso/portal-security-sso-opensso-impl/src/main/java/com/liferay/portal/security/sso/opensso/internal/OpenSSOImpl.java]: This endpoint was removed! https://backstage.forgerock.com/knowledge/kb/article/a54451431

**Comment by topolik** [comment]
Hi Marta, please resend. Other than those things it looks good!

**Comment by topolik** [comment]
And please rebase, there was a conflict in the portal-settings-lang properties :(

**Comment by topolik** [comment]
View total diff: https://github.com/topolik/liferay-portal/compare/28b4872d07...39d684ddea

File: modules/apps/portal-security-sso/portal-security-sso-opensso-impl/src/main/java/com/liferay/portal/security/sso/opensso/internal/OpenSSOImpl.java (hunk 1 above)
Comment (martamedio) [reply modules/apps/portal-security-sso/portal-security-sso-opensso-impl/src/main/java/com/liferay/portal/security/sso/opensso/internal/OpenSSOImpl.java]: @topolik about this... I reviewed it to see how to implement it and it turns out that it is not true that it's removed, it's working on OpenAM 12 and OpenAM 13, always returns a correct response (200) and the cookie value :confused:

openam13-getcookienamestoforward

Comment (topolik) [reply modules/apps/portal-security-sso/portal-security-sso-opensso-impl/src/main/java/com/liferay/portal/security/sso/opensso/internal/OpenSSOImpl.java]: Oh, hm, ok then.

**Pull request closed without merging**